app/src/main/res/layout/item_country_report.xml

```diff
@@ -233,9 +233,10 @@
                     android:id="@+id/bookmark_button"
                     android:layout_width="30dp"
                     android:layout_height="30dp"
+                    android:layout_marginStart="95dp"
```

Comment:
It seems to break center position of the icons, why is it added here?

Reply:
removed.